src/Maestro/Maestro.Web/Api/v2018_07_16/Controllers/AssetsController.cs

```diff
@@ -78,5 +80,68 @@ public async Task<IActionResult> GetAsset(int id)
 
             return Ok(new Asset(asset));
         }
+
+        [HttpPost("{assetId}/locations")]
+        [SwaggerResponse((int)HttpStatusCode.Created, Type = typeof(Channel))]
```

Comment:
This needs to match the status code and type of the return value.

Reply:
will fix.